index.bs

```diff
@@ -179,20 +181,19 @@ interface mixin UnidirectionalStreamsTransport {
          1. The |transport|'s state transitions to `"closed"` or `"failed"`.
          1. |p| has not been [=settled=].
 
-: <dfn for="UnidirectionalStreamsTransport" method>receiveStreams()</dfn>
-:: Returns a {{ReadableStream}} of {{ReceiveStream}}s that have been received
-   from the remote host.
+: <dfn for="UnidirectionalStreamsTransport" attribute>incomingUnidirectionalStreams</dfn>
+:: A {{ReadableStream}} of unidirectional streams, each represented by a
+   {{ReceiveStream}} object, that have been received from the remote host."
```

Comment:
Sorry, I'm confused why this line ends in a `"`.

Reply:
A typo.  Fixed.